Deprecation warning with Qt 5.15

```
/home/pmos/build/src/ubuntu-download-manager-6d1d305da3f6e9c3bfba278fb7268b80c48ef7fd/src/common/public/ubuntu/transfers/metadata.cpp: In member function 'QVariantMap Ubuntu::Transfers::Metadata::custom() const':
/home/pmos/build/src/ubuntu-download-manager-6d1d305da3f6e9c3bfba278fb7268b80c48ef7fd/src/common/public/ubuntu/transfers/metadata.cpp:217:48: error: 'QList<T> QMap<K, V>::values(const Key&) const [with Key = QString; T = QVariant]' is deprecated: Use QMultiMap for maps storing multiple values with the same key. [-Werror=deprecated-declarations]
  217 |             custom.insert(customKey, values(key)[0]);
      |                                                ^
In file included from /usr/include/QtCore/QMap:1,
                 from /home/pmos/build/src/ubuntu-download-manager-6d1d305da3f6e9c3bfba278fb7268b80c48ef7fd/src/common/public/ubuntu/transfers/metadata.h:22,
                 from /home/pmos/build/src/ubuntu-download-manager-6d1d305da3f6e9c3bfba278fb7268b80c48ef7fd/src/common/public/ubuntu/transfers/metadata.cpp:22:
/usr/include/QtCore/qmap.h:1356:10: note: declared here
 1356 | QList<T> QMap<Key, T>::values(const Key &key) const
      |          ^~~~~~~~~~~~
cc1plus: all warnings being treated as errors
```
I'll likely fix it quite soon

--- a/src/common/public/ubuntu/transfers/metadata.cpp
+++ b/src/common/public/ubuntu/transfers/metadata.cpp
@@ -214,7 +214,7 @@ Metadata::custom() const {
         if (key.startsWith(CUSTOM_PREFIX)) {
             QString customKey = key;
             customKey.replace(CUSTOM_PREFIX, "");
-            custom.insert(customKey, values(key)[0]);
+            custom.insert(customKey, value(key));
         }
     }
     return custom;

